src/pages/about-us.astro

```diff
@@ -13,28 +13,34 @@ import { resolvePath } from '@/utils/helpers.js'
       <div class="c-main-wrapper" id="header1">
         <div class="is-posttitle">ABOUT GROUP INCOME</div>
         <h1 class="is-title-1 c-main-title">On a mission to ensure<span class="text-orange"> basic</span><span class="text-green"> income</span> security for<span class="text-blue"> everyone</span>.</h1>
-        <p class="is-subtitle c-main-para">Group Income is an open-source project of the OkTurtles Foundation, a non-profit organization that supports beneficial decentralization technologies.</p>
+        <p class="is-subtitle c-main-para">Group Income is an open-source project of the <a class="is-link" href="https://okturtles.org" target="_blank">okTurtles Foundation</a>, a non-profit organization that supports beneficial decentralization technologies.</p>
       </div>
     </section>
     <section class="container span-container c-container">
       <div class="span-2"><img class="c-container-image" src={resolvePath("/images/about/graph1.png")} /></div>
       <div class="span-2">
         <h2 class="is-title-2">Open Protocol</h2>
-        <p class="c-container-para">Our software is non proprietary and is not owned by any particular company. The protocols in the Internet stack are open so that any computing device can follow the protocol to join the global network.</p>
+        <p class="c-container-para">
+          Group Income is built on top of an <a class="is-link" href="https://shelterprotocol.net" target="_blank">open protocol</a>, and Group Income itself is 100% open-source (both the frontend and the backend). This means anyone can inspect our code, and anyone can create their own version of Group Income that interacts with ours.
```

Comment:
I'm not sure about this `class="is-link"` thing.

Doesn't it seem a bit odd that the majority of links — the default link styling — needs a special class?

Shouldn't it be the other way around: that special cases need special classes, but otherwise all `a` tags use the same default styling?

Not doing it that way leads to problems like this (from this PR):

![Screenshot 2024-03-26 at 5 33 13 PM](https://github.com/okTurtles/groupincome.org/assets/138706/29deee80-47ea-4b1c-9441-147b171f49c6)

"federated protocol" is a link there, but it's not highlighted because `class="is-link"` was forgotten on it.

Could you make the `is-link` class styling the default for all links, and then wherever we have special links to which the defaults don't apply, there we can have special classes used?

Reply:
@taoeffect 
That was actually my initial approach, thinking it would make the job easier. But then I realised applying the styles to `a` tag leads to a bunch of unwanted issues like below:

1]
<img src='https://github.com/okTurtles/groupincome.org/assets/17641213/a9c230bf-cdfe-4c00-b140-52343cf80305' width='430'>

2]
<img src='https://github.com/okTurtles/groupincome.org/assets/17641213/e90a67d0-78e3-4125-8e97-c019d8169885' width='430'>

3]
<img src='https://github.com/okTurtles/groupincome.org/assets/17641213/e16484d5-f799-4c6e-bac1-e3f258bd9e84' width='430'>

4]
<img src='https://github.com/okTurtles/groupincome.org/assets/17641213/efaf15f7-9624-4e65-84d1-eb28454c708f' width='430'>

And there is more actually.

---
meaning it creates a different bunch of bugs depending on the context the a tag is used. (One example is, if the container element has `display: flex; flex-direction: column;` with `align-items: stretch;`, it ends up making the underline go beyond the actual text)

After realising this, I decided that specifying `.is-link` class selectively would be the best approach.

Reply:
@taoeffect Fixed this issue you posted btw:

<img src='https://github.com/okTurtles/groupincome.org/assets/17641213/3b1217c0-689b-42ab-bc8e-3d03c8a305d4' width='430'>